src/lambda/templates/errorTemplates.ts

```diff
@@ -0,0 +1,34 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export class ErrorTemplates {
+    public static readonly SHOW_ERROR_DETAILS = `
+    <h1>
+        Error details for <%= parent.label %>
```

Comment:
Any English text needs to be localized, so the entire text `Error details for {0}` should be localized.

The same applies for `Error code`, `Error message`, and `Stack trace`.